CHANGELOG.md

```diff
@@ -1,5 +1,7 @@
 # Changelog
-### 1.2.3 (In Progress)
+### 1.2.5 (In Progress)
```

Comment:
Why are we skipping 1.2.4?

Reply:
Its in the additional notes of the PR
"Please note that the changelog has been modified to use the version 1.2.5 for the in-progress version as I plan on making a release to test that the issue is fixed."